tape/src/test/java/com/squareup/tape/ObjectQueueTest.java

```diff
@@ -14,110 +15,103 @@
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
+import org.junit.runner.RunWith;
 
 import static org.fest.assertions.Assertions.assertThat;
 import static org.fest.assertions.Fail.fail;
 
-public class FileObjectQueueTest {
+@RunWith(BurstJUnit4.class)
+public class ObjectQueueTest {
+  public enum QueueFactory {
+    FILE() {
+      @Override public <T> ObjectQueue<T> create(File file, FileObjectQueue.Converter converter)
+          throws IOException {
+        return new FileObjectQueue<T>(file, converter);
+      }
+    },
+    MEMORY() {
+      @Override public <T> ObjectQueue<T> create(File file, FileObjectQueue.Converter converter) {
+        return new InMemoryObjectQueue<T>();
+      }
+    };
+
+    public abstract <T> ObjectQueue<T> create(File file, FileObjectQueue.Converter converter) throws
+        IOException;
+  }
+
   @Rule public TemporaryFolder folder = new TemporaryFolder();
-  private FileObjectQueue<Pojo> queue;
+  @Burst QueueFactory factory;
+  ObjectQueue<String> queue;
```

Comment:
Should we keep using a `Pojo` type here? Didn't see big upside/downside of doing so.